source/fundamentals/data-formats/pojo-customization.txt

```diff
@@ -419,24 +415,19 @@ code example:
 
 If a database client updated the documents in the collection to include
 additional fields that describe the products such as ``dimensions`` and
-``weight``, a document retrieved using the ``Product`` POJO might resemble
-the following JSON object:
+``weight``, the document retrieved using the ``Product`` POJO might resemble
```

Comment:
I still feel like it would aid clarity if you show the JSON exactly instead of just describing it.  The way it is, there is room for misinterpretation.